pkgs/development/libraries/libtickit/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libtermkey
+, unibilium
+, libtool
+}:
+stdenv.mkDerivation rec {
+  pname = "libtickit";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "leonerd";
+    repo = "libtickit";
+    rev = "v${version}";
+    hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

```suggestion
  makeFlags = [
    "PREFIX=$(out)"
    "LIBTOOL=${lib.getExe libtool}"
  ];
```
It seems to try to run `glibtool` and it fails on darwin.
https://github.com/leonerd/libtickit/blob/5d006987078515f0768259856abc9aad20551d76/Makefile#L1-L2